drivers/can/Kconfig.net

```diff
@@ -0,0 +1,38 @@
+# Kconfig.net - Configuration options for IPv6 over CAN
+
+#
+# Copyright (c) 2019 Alexander Wachter
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+config CAN_NET
+	bool "Enable 6loCAN network interface [Experimental]"
+	default n
+	help
+	  Enable IPv6 Networking over can (6loCAN)
+
+if CAN_NET
+
+module = CAN_NET
+module-dep = NET_LOG
+module-str = Log level for Network CAN
+module-help = Enables logging for CAN L2 networking
+source "subsys/net/Kconfig.template.log_config.net"
+
+config CAN_NET_NAME
+	string "Network interface name"
```

Comment:
Isn't this the device name as typically our network interfaces do not have a name.

Reply:
Changed to `Network device name`